README.md

```diff
@@ -7,107 +7,20 @@
 
 istio-csr is an agent that allows for [Istio](https://istio.io) workload and
 control plane components to be secured using
-[cert-manager](https://cert-manager.io). Certificates facilitating mTLS, inter
-and intra cluster, will be signed, delivered and renewed using [cert-manager
+[cert-manager](https://cert-manager.io). Certificates facilitating mTLS - both inter
+and intra cluster - will be signed, delivered and renewed using [cert-manager
```

Comment:
I could not resist, sorry!! em-dashes 😅


as Michael Kashin would put it, "my life will never be the same."
<a href="https://twitter.com/networkop1/status/1465932140005269504"><img alt="" src="https://user-images.githubusercontent.com/2195781/145618207-f9d0be9f-650a-4d6e-9c66-0f1bf9c83006.png" width="300"></a>

```suggestion
[cert-manager](https://cert-manager.io). Certificates facilitating mTLS — both inter
and intra-cluster — will be signed, delivered and renewed using [cert-manager
```